rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -323,6 +323,67 @@ public void handmade_release_after_rskip_146_rejected_contractCaller() throws IO
         }
     }
 
+    @Test
+    public void release_before_rskip_219() throws IOException {
```

Comment:
```suggestion
    public void release_after_rskip_219() throws IOException {
```
You got the names mixed-up